README.md

```diff
@@ -194,143 +182,145 @@ Filenames or directory paths | Outputs documentation for only specified files or
 `--help` | Displays help documentation for the command. | Markdown, JSON    | If not specified, Strings outputs HTML documentation.
 
-## Documenting Puppet code for Strings
 
-Strings relies on code comments and YARD docstrings to specify documentation comments. You can also include any number of YARD tags that hold semantic metadata for various aspects of the code. These tags allow you to add this information to your module without worrying about presentation.
+# Puppet Strings style
 
-### Documenting Puppet classes and defined types
+Applies to Puppet 4 and later
 
-To document Puppet classes and defined types, use a series of comments to create a YARD docstring before the class or defined type definition.
+Puppet Strings combines source code and code comments to create complete, user-friendly reference information for modules. Strings can generate module documentation for classes, defined types, functions, and resource types in HTML, JSON, and Markdown formats.
 
-```puppet
-# @summary A short summary of the purpose of the class.
-#
-# @example Declaring the class
-#   include example
-#
-# @param first The first parameter for this class
-# @param second The second parameter for this class
-#
-class my_class(
-  String $first  = $my_class::params::first_arg,
-  Integer $second = $my_class::params::second_arg,
-) inherits my_class::params {
-  # ...
-}
-```
+Instead of manually writing and formatting long reference lists, add a few descriptive tags and comments for each element (class, defined type, function) of your module. Whenever you update code, update your documentation comments at the same time. Strings automatically extracts some information, such as data types and attribute defaults from the code, so you need to add minimal documentation comments.
 
-The Strings elements appearing in the above comment block are:
 
-* The `@summary` tag, a short description of the class (should be fewer than 140 characters).
-* The `@example` tag, immediately followed by an optional title.
-* Under the `@example` tag, indented two spaces, the usage example code.
-* Two `@param` tags, with the name of the parameter first, followed by a string describing the parameter's purpose.
+## Module README
 
-Puppet Strings automatically documents information such as data types, default values, the namevar, and the return value type for functions.
+Module READMEs are where users can learn more about what a module does and how to use it. In the module README, include basic module information and extended usage examples that address common use cases. 
 
-Defined types are documented in exactly the same way as classes:
+Complete information about classes, defined types, functions, and resource types are generated by Strings and output in HTML, Markdown, or JSON format. Strings does not yet generate information for type aliases, facts, or custom providers. If your module includes these elements, document them in your README.
```

Comment:
Should this be "Complete information about classes, defined types, functions, and resource types _is_ generated..."?

Also, it turns out we DO pull in custom providers. Hope that's not too much of a wrench in the works this late in the game.

Reply:
D'OH